src/doc/reference.md

```diff
@@ -1730,7 +1730,8 @@ of an item to see whether it should be allowed or not. This is where privacy
 warnings are generated, or otherwise "you used a private item of another module
 and weren't allowed to."
 
-By default, everything in Rust is *private*, with one exception. Enum variants
+By default, everything in Rust is *private*, with two exceptions: Associated
+items of a Trait are public by default; Enum variants
```

Comment:
Associated items and variants are not public by default, they inherit visibility from their traits and enums respectively. E.g.
```
pub(vis) trait Tr {
    fn f();
}
```
is "desugared" into
```
pub(vis) trait Tr {
    pub(vis) fn f(); // (Not syntactically legal)
}
```